launch/driver.launch

```diff
@@ -4,9 +4,10 @@ Licensed under the MIT License.
 -->
 
 <launch>
+  <arg name="tf_prefix"         default="" />               <!-- Prefix added to tf frame IDs. It typically contains a trailing '_' unless empty. -->
 
   <param name="robot_description"
-    textfile="$(find azure_kinect_ros_driver)/urdf/azure_kinect.urdf" />
+    command="$(find xacro)/xacro.py $(find azure_kinect_ros_driver)/urdf/azure_kinect.urdf.xacro tf_prefix:=$(arg tf_prefix)" />
```

Comment:
On startup, I get a warning:

```
xacro.py is deprecated; please use xacro instead
```

Changing to just `xacro` instead of `xacro.py` does indeed make the warning go away. Can you change this to just `xacro`?

Reply:
done